### DISCUSSION TITLE
Missing restart field (CAPE_old_dCAPEd) in E3SMv2 branch from 0501-01-01 year piControl

### DISCUSSION THREAD

**Discussion by jacob-stu-allen** [E3SM-Project/E3SM#5768] [E3SM model help]
Hello!

I ran a "hybrid" case from the piControl run successfully, but I was hoping to use the "branch" option for the next simulations. I've got the restart files from the "v2_1.LR.piControl" staged into the run directory, but the model hasn't been able to start due to the following error: 

ERROR: (PIO:   -49) read_restart_field_double: CAPE_old_dCAPEd not found

Running ./check_input_data in my case_scripts directory seems to result in no problems, but I'm also not seeing the "CAPE_old_dCAPEd" variable anywhere in the EAM restart file (I assume this is where that would live?).

Thanks in advance for any help,
Jacob

**Comment by jacob-stu-allen** [comment]
Hello again - just commenting on this topic to hopefully bring it back into view.

**Comment by jacob-stu-allen** [comment]
This has been fixed - the restart files were from E3SMv2.1 and I had cloned the E3SMv2.0 code instead of 2.1, which was not happy with the restarts I had been feeding it.

**Comment by rljacob** [comment]
Glad you figured it out.  Restarts are not transferrable from version to version.
